### DISCUSSION TITLE
Subsequent property declarations must have the same type.

### DISCUSSION THREAD

**Discussion by naturalstyl** [l-lin/angular-datatables#1741] [Q&A]
Hello. I'm trying to resolve some vulnerabilities in my Angular application. So I recently updated datatables.net (^1.13.4) to use with angular-datatables (^11.2.0). Now I'm getting the following error when the app is compiled.
```
Subsequent property declarations must have the same type.  Property 'DataTable' must be of type '(opts?: 
Settings) => Api', but here has type 'JQueryDataTableApi'

 node_modules/@types/datatables.net/index.d.ts:16:5
    16     DataTable(opts?: DataTables.Settings): DataTables.Api;
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'DataTable' was also declared here.
```
How do I resolve this and which import is angular-datatables is using? I've attached my list of npm packages.
![Declare_Databtables_Confusion-2](https://github.com/l-lin/angular-datatables/assets/13439142/d5a62f33-c58b-4e4e-8587-63e1d2deb8ca)

**Comment by shanmukhateja** [comment]
Hi,

I think datatables.net provides their own typings, can you uninstall `@types/datatables.net` package? 

Also, its not uncommon for @types packages to end up broken every couple of months. This is why we suggest using any type for advanced use cases.
